docs/facts/examples.ipynb

```diff
@@ -118,6 +119,7 @@
     "    {\n",
     "        \"view\": [\n",
     "            {\n",
+    "                \"coordinate\": \"cartesian\",\n",
```

Comment:
This should not be required to be set manually, no?

Reply:
This is not required to be set manually. The recommender can pick based on soft constraints. I just wanna add to examples to clarify which coordinate system the chart is using.

Reply:
I would prefer to remove unnecessary details.